website-react/src/models/SearchModel.js

```diff
@@ -38,7 +39,9 @@ export default mirror.model({
     async search (searchTerm) {
       actions.search.startLoading()
 
-      const url = config.baseUrl.replace('{searchTerm}', searchTerm)
+      const url = config.baseUrl
+        .replace('{searchTerm}', searchTerm)
+        .replace('{maxResults}', config.maxResults)
```

Comment:
Would have liked to get the maxResults value from the store at this point - but doesn't seem like mirror offers that, and I don't know if I want it bad enough that I want to require the search component to pass it in as argument. That could easily be done though, if we want it to be user-configurable what the search result limit is.